pkg/nss/certutilservice.go

```diff
@@ -11,3 +11,31 @@ package nss
 //  the CertUtilService struct.
 
 var _ CertUtilServicer = &CertUtilService{}
+
+// CertUtilService encapsulates operations on NSS database using certutil command.
+type CertUtilService struct {
+	runner Runner
+}
+
+// NewCertUtilService returns a new CertUtilService.
+func NewCertUtilService(runner Runner) (*CertUtilService, error) {
+	return &CertUtilService{runner: runner}, nil
+}
+
+// AddCA adds a certificate to the NSS database CA list.
+func (s *CertUtilService) AddCA(dbPath string, certificateName string, certPath string) error {
+	//nolint:wrapcheck
+	return s.runner.Run("-A", "-d", dbPath, "-t", "C,,", "-n", certificateName, "-i", certPath)
```

Comment:
ho it's the same parameters for certutils and mar-tools!

Reply:
In fact, `mar-tools` embeds `certutils`, so, we are using the same binary (at lest, they work the same way)